web/views.py

```
@@ -7018,3 +7021,97 @@ def users_list(request: HttpRequest) -> HttpResponse:
     }
 
     return render(request, "users_list.html", context)
+
+
+@login_required
+def topic_vote(request, pk):
```

Ruff (0.8.2)

Missing return type annotation for public function `topic_vote`

(ANN201)